db/schema.rb

```diff
@@ -67,17 +74,7 @@
     t.index ["name"], name: "index_groups_on_name", unique: true
   end
 
-  create_table "help_texts", force: :cascade do |t|
-    t.bigint "service_provider_id"
-    t.json "sign_in", default: {}
-    t.json "sign_up", default: {}
-    t.json "forgot_password", default: {}
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.index ["service_provider_id"], name: "index_help_texts_on_service_provider_id"
-  end
```

Comment:
Per previous PR, this table may have never been committed as a migration, it's no longer in git history